src/pages/loginPage/Login.tsx

```diff
@@ -0,0 +1,325 @@
+  width: 376px;
+  height: 24px;
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: rgba(0, 0, 0, 0.87);
+`;
+
+const Login = () => {
+  const navigate = useNavigate();
+
+  const formSchema = Yup.object().shape({
+    email: Yup.string()
+      .required('Email không được bỏ trống')
+      .email('Email không hợp lệ'),
+    password: Yup.string()
+      .required('Mật khẩu không được bỏ trống')
+      .min(8, 'Mật khẩu phải có ít nhất 8 ký tự')
+      .trim()
+      .matches(/^\S*$/, 'Mật khẩu không được có khoảng trắng')
+  });
+
+  const validationOpt = {
+    resolver: yupResolver(formSchema)
+  };
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors }
+  } = useForm<Inputs>(validationOpt);
+
+  const onSubmit = () => {
+    navigate('/user');
+  };
+
+  const forgotPasswordPage = () => {
+    navigate('/forgot-password');
+  };
+
+  const registerPage = () => {
+    navigate('/register');
+  };
+
+  return (
+    <LoginPage>
+      <LoginPageInside>
+        <SideLeft>
+          <ImgLeft src={img1} alt="" />
+        </SideLeft>
+        <SideRight>
+          <ContainerRight>
+            <Header>
+              <Typography
+                variant="h4"
+                align="center"
+                sx={{
+                  width: '376px',
+                  height: '42px',
+                  fontFamily: 'Roboto',
+                  fontStyle: 'normal',
+                  fontWeight: '700',
+                  fontSize: '33px',
+                  lineHeight: '123.5%',
+                  color: 'rgba(0, 0, 0, 0.87)'
+                }}>
+                Đăng nhập vào tài khoản
+              </Typography>
+            </Header>
+            <Form onSubmit={handleSubmit(onSubmit)}>
+              <BoxEmail>
+                <Label htmlFor="email">Email</Label>
+                <TextField
+                  {...register('email')}
+                  helperText={errors.email?.message && errors.email.message}
+                  type="text"
+                  id="email"
+                  placeholder="Email: Nguyễn Văn A"
+                  sx={{
+                    width: '100%'
+                  }}
+                  required
+                />
+              </BoxEmail>
+              <BoxPassword>
+                <Label htmlFor="password">Mật khẩu</Label>
+                <TextField
+                  {...register('password')}
+                  helperText={
+                    errors.password?.message && errors.password.message
+                  }
+                  type="password"
+                  id="password"
+                  placeholder="Mật Khẩu"
+                  sx={{
+                    width: '100%'
+                  }}
+                  required
+                />
+              </BoxPassword>
+              <Links onClick={forgotPasswordPage}>
+                <span>Quên mật khẩu?</span>
+              </Links>
```

Comment:
Chú ý khi dùng Link thì dùng component Link, nó sẽ render ra thẻ a
Về mặt UX thẻ a là được đề nghị sử dụng khi routing, nó sẽ có các default behavior của trình duyệt, và thân thiện với người dùng hơn. (Ví dụ có thể open in new tab khi chuột phải hoặc chuột giữa, visited link,...)
Trường hợp bất khả kháng hoặc đặc thù mới dùng navigate
Tham khảo cách dùng Link component và tương tác với react-router https://mui.com/material-ui/guides/routing/#link